sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/IngestionRedirectPolicy.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable // TODO: remove and fix errors
+
+using Azure.Core;
+using System.Threading.Tasks;
+using System;
+using Azure.Core.Pipeline;
+using System.Net.Http.Headers;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal sealed class IngestionRedirectPolicy : HttpPipelinePolicy
+    {
+        internal const int MaxRedirect = 10;
```

Comment:
could be helpful to add a comment saying it is as per telemetry collection spec (not sure if can add a link to it here)